src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
Thanks @hellobrian for following this up! The `.bx--breadcrumb--no-trailing-slash .bx--breadcrumb-item:last-child::after` selector looks good to me.